Makefile

```diff
@@ -30,6 +30,10 @@ core-download:
 	wget https://downloads.dcos.io/cli/plugins/dcos-core-cli/1.12/darwin/x86-64/dcos-core-cli.zip -O build/darwin/core.zip
 	wget https://downloads.dcos.io/cli/plugins/dcos-core-cli/1.12/windows/x86-64/dcos-core-cli.zip -O build/windows/core.zip
 
+.PHONY: install
+install: generate
```

Comment:
For consistency with the "go build" target, can we make this one not depend on "generate" and accept the build tags from the env var too?